CSS/WorkPg.css

```diff
@@ -0,0 +1,115 @@
+.worksection {
+  margin: 24px;
+}
+
+.workheading {
+  text-align: center;
+}
+
+.gymimg {
+  margin-top: 80px;
+  margin-right: none;
+  position: static;
+  width: 527px;
+  height: 437px;
+  top: 0;
+  left: 24px;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] Kindly the fixed width and height here `width: 527px;
  height: 437px;` make your page have a horizontal scrolling because of this image. Kindly delete them and it will look perfect.

![image](https://user-images.githubusercontent.com/35535130/166122844-872f3167-7eaf-488c-bf43-319acb299353.png)